base_location/__manifest__.py

```diff
@@ -3,32 +3,27 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    'name': 'Location management (aka Better ZIP)',
-    'version': '13.0.1.0.0',
-    'depends': [
-        'base_address_city',
-        'contacts',
+    "name": "Location management (aka Better ZIP)",
```

Comment:
Can we avoid this quote change?

Reply:
Black has an option to not do string normalization yes. Personally I'm in favor of letting it normalize, since it's one less thing to think about.

Reply:
Well, I prefer not, as this change doesn't add anything valuable for homogenization (it looks pretty the same), and it can generate big diffs (as this case) and also the need to escape `"` characters inside the string.

Reply:
Thanks for this info, Yannick